<!DOCTYPE html>
<html>
<head>
  <title>Jason Zhu</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <link rel="stylesheet" href="css/bootstrap.min.css" />
  <link rel="stylesheet" href="css/style.css" />
  <script type="text/javascript" src="js/jquery.min.js"></script>
  <script type="text/javascript" src="js/bootstrap.min.js"></script>
</head>
<body>
  <div class="container main_container" id="header_container">
    <div class="jumbotron" id="aegis_logo">
      <div class="face-name">
        <div>
          <img src="img/headshot.png" class="img-headshot">
        </div>
        <div class="title">
          <h1>Junjie (Jason) Zhu</h1>
          <!-- <br> -->
          <p style="line-height: 100px;"></p>
          <p>Ph.D. Candidate, Electrical Engineering</p>
          <p>M.S. in Statistics, Stanford University</p>
          <p><a>jjzhu at stanford dot edu</a></p>
          <!-- <br> -->
          <p>
            <a href="docs/CV/jzhu_cv.pdf">CV (07/02/2019)</a>,
            <a href="https://scholar.google.com/citations?user=2EasRdEAAAAJ&hl=en">Google Scholar Profile</a>,
            <a href="https://www.linkedin.com/in/jason-junjie-zhu">LinkedIn Profile</a>
          </p>
        </div>
      </div>
    </div>
  </div>
  <div class="container main_container" id="story_container">
    <div class="outertextbox">
      <div class="textbox">
        <p>
          <i>
            Trained as an engineer, statistician, and computational biologist, I am passionate about providing principled solutions to analyzing complex data sets. In particular, I am interested in utilizing the power of <a href="https://mitpress.mit.edu/books/atlas-science">mapping</a> to understand large-scale biological systems and scientific knowledge bases.
          </i>
        </p>
      </div>

    </div>
  </div>
  <div class="container main_container" id="content_container">
   <div aria-multiselectable="true" role="tabl ist" id="accordion" class="panel-group">
    <div class="panel panel-default"> <a title="Tab 1" aria-controls="collapse1" aria-expanded="true" href="#collapse1" data-parent="#accordion" data-toggle="collapse" id="heading1" role="tab" class="panel-heading"><span class="glyphicon icon-indicator"></span> <span class="panel-title">About Me</span> </a>
      <div aria-labelledby="heading1" role="tabpanel"
        class="panel-collapse collapse in"
        id="collapse1"
        aria-expanded="true">
        <div class="panel-body">
          <p>I am a Ph.D. student at
            <a href="http://www.stanford.edu/">Stanford</a>
            working in statistics and machine learning with applications in biology and medicine. My main interests are the computational and statistical challenges presented by high-throughput genomics data. I am supervised by Professor
          <a href="http://statweb.stanford.edu/~sabatti/">Chiara Sabatti</a>
          in
          <a href="https://statistics.stanford.edu/">Statistics</a>
          and
          <a href="http://med.stanford.edu/dbds.html">Biomedical Data Science</a>.
          </p>
          <p>
          <b>My expertise can be broadly categorized into three domains:</b>
          <ul>
            <li>high-dimension statistics where the goal is to simultaneously test a large number of hypotheses</li>
            <li>unsupervised learning where the goal is to cluster and visualize unlabeled data for pattern mining</li>
            <li>network science where the goal is to generate and interpret relational structures from the data</li>
          </ul>
          </p>
          <p> My projects are highly interdisciplinary. I have also had multiple long-term collaborations with Professor
          <a href="http://www.serafimb.org/">Serafim Batzoglou</a> in
          <a href="http://ai.stanford.edu/">Computer Science</a> and Professor
          <a href="http://kuolab.stanford.edu/">Calvin Kuo</a>
          in
          <a href="http://med.stanford.edu/">Medicine</a>.
          I have been fortunate to apply my training in both engineering and statistics to new biotechnologies during my internships at
          <a href="https://www.10xgenomics.com/">10x Genomics</a> and
          <a href="https://www.illumina.com/">Illumina</a>.
          </p>

          <p>I received my M.S. degree in Statistics from Stanford University and my B.S. degree in Electrical and Computer Engineering from
          <a href="http://www.olin.edu/">Olin College of Engineering</a>, where I worked with
          <a href="http://www.olin.edu/faculty/profile/siddhartan-govindasamy/">Siddhartan Govindasamy</a>,
          <a href="http://olin.edu/faculty/profile/sarah-s-adams/">Sarah Spence Adams</a>, and
          <a href="http://www.babson.edu/Academics/faculty/profiles/Pages/Troxell-Denise.aspx">Denise Troxell</a>.
          During my undergraduate studies, I was also a visiting research student with Professor
          <a href="http://www.mckayspcb.com/">Matthew McKay</a>
          at
          <a href="http://www.ust.hk/">Hong Kong University of Science and Technology</a>.
          </p>
        </div>
      </div>
    </div>
    <div class="panel panel-default"> <a title="Tab 2" aria-controls="collapse2" aria-expanded="false" href="#collapse2" data-parent="#accordion" data-toggle="collapse" id="heading2" role="tab" class="panel-heading collapsed"><span class="glyphicon icon-indicator"></span> <span class="panel-title">Current Research</span> </a>
        <div aria-labelledby="heading2" role="tabpanel" class="panel-collapse collapse" id="collapse2" aria-expanded="false">
            <div class="panel-body">
                <p>
                  These are my current Ph.D. projects in bioinformatics and biostatistics at Stanford. I have had the good fortune to have many fruitful collaborations in academia and in industry. These projects would not have been possible without my extermely talented and hard-working collaborators.
                </p>

                <div class="project">
                  <div class="proj_row">
                    <div class="proj_txt">
                      <h5>Gene Ontology and Graph Visualization</h5>
                      <p>
                        The Gene Ontology (GO) is one of the most popular biological knowledge databases for discovery-based omics research. Yet, the complexity of its data structure, represented as a large Directed Acyclic Graph (DAG) with tens of thousands of nodes, and its continuously evolving annotations can limit our interpretation and exploration in many applications.
                      <p>
                        I designed new algorithms and developed a full-stack software tool (AEGIS) to visualize the GO within an interactive framework. AEGIS overcomes the limitation of existing layout strategies that determine GO term specificity in the hierarchy by integrating both the GO structure and gene annotations into a new DAG layout. I also proposed an interactive visualization tailored for large DAGs by devising a focus-and-context framework, which can be used to navigate the DAG on a fine-grained level, while preserving contextual information of the entire DAG.
                      </p>
                      <p>
                        <a href="http://aegis-viz.appspot.com/">Website</a>:
                        AEGIS and its visualization applications
                        <br/>
                        <a href="https://www.biorxiv.org/content/early/2018/10/05/436741">Paper</a>: Methodology behind AEGIS
                        <br/>
                        <a href="docs/cs448b_final_report.pdf">Report</a>: Project on D3 visualization
                      </p>
                    </div>
                    <div class="proj_img">
                      <h5>The Map of Biological Concepts</h5>
                      <img src="img/map_go.png" class="proj_fig">
                    </div>
                  </div>
                  <div class="proj_row">
                    <div class="proj_txt">
                      <h5>Single-Cell Genomics and Systems Biology</h5>
                      <p>
                        Single-cell technologies are a powerful means to measure gene expression levels of individual cells and to reveal previously unknown heterogeneity and functional diversity among cell populations. However, unknown biological variability and high measurement noise can make it difficult to analyze single-cell data.
                      </p>
                      <p>
                        Across multiple projects, I have developed new unsupervised learning (dimension reduction and clustering) techniques that are robust across sequencing platforms, analyzed over 50 data sets to build a comprehensive database, and contributed to a number of computational and statistical pipelines for new biotechnologies and applications. A lot of our techniques also extend to other biological data analysis. I have worked on single-cell applications that span immunology, oncology, and stem cell biology.
                      </p>
                      <p>
                        <a href="https://www.sciencedirect.com/science/article/pii/S0092867418315137">Paper 1</a>: The tumor immune microenvironment
                        <br/>
                        <a href="https://www.sciencedirect.com/science/article/pii/S0092867418315046">Paper 2</a>: Long-term expansion of primary hepatocytes
                        <br/>
                        <a href="https://www.nature.com/articles/nmeth.4207">Paper 3</a>: SIMLR: kernel-based similarity learning
                        <br/>
                        <a href="http://www.mdpi.com/2073-4425/8/12/368/html">Paper 4</a>:
                        Single-cell RNA-seq database
                        <br/>
                        <a href="https://www.nature.com/articles/ncomms14049">Paper 5</a>: 10x Genomics droplet technology
                        <br/>
                        <a href="http://papers.nips.cc/paper/6290-unsupervised-learning-from-noisy-networks-with-applications-to-hi-c-data">Paper 6</a>: Extended application to Hi-C data
                        <br/>
                        <a href="https://github.com/BatzoglouLabSU/SIMLR">Code 1</a>: R, Python and MATLAB code for SIMLR
                        <br/>
                        <a href="https://github.com/10XGenomics/single-cell-3prime-paper">Code 2</a>: R scripts for 10x droplet technology
                      </p>
                    </div>
                    <div class="proj_img">
                      <h5>The Map of Single Cells</h5>
                      <img src="img/map_sincel.png" class="proj_fig">
                    </div>
                  </div>
                  <div class="proj_row">
                    <div class="proj_txt">
                      <h5>Genetic Regulation in Human Tissues</h5>
                      <p>
                      The Genotype-Tissue Expression (<a href="https://www.gtexportal.org/home/">GTEx</a>) project aims to discover associations between indivdual genetic variance and the gene expression of different tissues. It consists of 53 types of human tissues from healthy, postmortem donors who were densely genotyped. These tissues are costly to acquire, so one of the main challenges is detecting signals with a very limited sample size.
                      </p>
                      <p>
                        I developed systematic analysis pipelines to conduct expression quantitative trait loci (eQTLs) studies where the goal is to identify tissue-specific gene expression that is highly correlated with genetic mutation.s With my collaborators, we proposed a selective inference analysis workflow to account for selective biases when analyzing a large number of genes, which can provide accurate confidence interval coverages with limited sample sizes. Additionally, I worked on gene function prediction problems with the same dataset as part of a data mining course project.
                      </p>
                      <p>
                        <a href="https://arxiv.org/abs/1801.08686">Paper</a>: Selection-adjusted Effect-size Estimation
                        <br/>
                        <a href="docs/cs341_final_report.pdf">Report</a>: Project report on gene function prediction
                      </p>
                    </div>
                    <div class="proj_img">
                      <h5>The Map of Human Tissues</h5>
                      <img src="img/map_tissue.png" class="proj_fig">
                    </div>
                  </div>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default"> <a title="Tab 3" aria-controls="collapse3" aria-expanded="false" href="#collapse3" data-parent="#accordion" data-toggle="collapse" id="heading3" role="tab" class="panel-heading collapsed"><span class="glyphicon icon-indicator"></span> <span class="panel-title">Previous Projects</span> </a>
        <div aria-labelledby="heading3" role="tabpanel" class="panel-collapse collapse" id="collapse3" aria-expanded="false">
            <div class="panel-body">
              <p>
                These are some of my selected project prior to Stanford. I am grateful to my undergraduate mentors who patiently taught me everything from scratch and got me excited about research early on. This theoretical foundation built my intuition for many applied projects in other fields during my graduate studies.
              </p>
              <div class="proj_row">
                <div class="proj_txt">
                  <h5>Graph Theory</h5>
                  <p>
                    The L(2,1)-labeling of a graph is an extension of the classical graph coloring problem. It is motivated by assigning frequencies to transmitters in a radio network. By investigating a wide family of graphs and enumerating many example graphs, we found general labeling patterns and unique graph structures. In addition, we proved a conjecture in a related coloring problem.
                  </p>
                  <p>
                    <a href="https://www.sciencedirect.com/science/article/pii/S0166218X12004313">Paper 1</a>: Amalgamation Graphs
                    <br/>
                    <a href="https://link.springer.com/article/10.1007/s10878-013-9632-x">Paper 2</a>: Edge-path Replacement Graphs
                    <br/>
                    <a href="https://msp.org/involve/2015/8-3/p14.xhtml">Paper 3</a>: Injective Labeling of General Graphs
                    <br/>
                    <a href="https://www.sciencedirect.com/science/article/pii/S0166218X14002972">Paper 4</a>: Cartesian Products of Complete Graphs
                    <br/>
                    <a href="https://www.sciencedirect.com/science/article/pii/S0166218X14004272">Paper 5</a>: Generalized Flowers
                    <br/>
                  </p>
                </div>
                <div class="proj_img old_proj">
                  <h5>Integer Labeling on Graphs</h5>
                  <img src="img/graph_lab.png" class="proj_fig">
                </div>
              </div>
              <div class="proj_row">
                <div class="proj_txt">
                  <h5>Communication Theory</h5>
                  <p>
                    Using stochastic geometry for the theoretical analysis of wireless networks, we  statistically characterized the performance of multi-antenna receivers under more general network topologies than what was available at the time. In addition, analyzing the impact of carrier frequency offsets on multi-user systems that apply orthogonal frequency-division multiplexing, I derived analytical results that determine the impact of unsynchronized interferers.
                  </p>
                  <p>
                    <a href="https://ieeexplore.ieee.org/abstract/document/6364332/">Paper 1</a>: Non-homogenous Poisson Networks
                    <br/>
                    <a href="https://ieeexplore.ieee.org/abstract/document/6655046/">Paper 2</a>: Networks with Clustered Interferers
                    <br/>
                    <a href="https://ieeexplore.ieee.org/abstract/document/6883650/">Paper 3</a>: Multi-antenna OFDM Systems
                    <br/>
                    <a href="https://ieeexplore.ieee.org/abstract/document/6825859/">Paper 4</a>: Doubly Stochastic Networks
                  </p>
                </div>
                <div class="proj_img old_proj">
                  <h5>Stochastic Wireless Networks</h5>
                  <img src="img/comm_net.png" class="proj_fig">
                </div>
              </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default"> <a title="Tab 4" aria-controls="collapse4" aria-expanded="false" href="#collapse4" data-parent="#accordion" data-toggle="collapse" id="heading4" role="tab" class="panel-heading collapsed"><span class="glyphicon icon-indicator"></span> <span class="panel-title">Publications</span> </a>
        <div aria-labelledby="heading4" role="tabpanel" class="panel-collapse collapse" id="collapse4" aria-expanded="false">
            <div class="panel-body">
              <p>Only peer-reviewed papers are listed here. An extended list can be found at my <a href="https://scholar.google.com/citations?user=2EasRdEAAAAJ&hl=en">Google Scholar</a> page.</p>
                <ol>
                <li><p><b>J. Zhu</b>, Q. Zhao, E. Katsevich, C. Sabatti,
                <a href="https://www.biorxiv.org/content/early/2018/10/05/436741">&ldquo;Exploratory gene ontology analysis with interactive visualization,&rdquo;</a>
                <i>under review</i>.
                </p>
                </li>
                <li><p>S. Panigrahi, <b>J. Zhu</b>, C. Sabatti,
                <a href="https://arxiv.org/abs/1801.08686">&ldquo;Selection-adjusted inference: an application to confidence intervals for cis-eQTL effect sizes,&rdquo;</a>
                <i>under review</i>.
                </p>
                </li>
                <li><p>
                J. T. Neal<sup>&dagger;</sup>, X. Li<sup>&dagger;</sup>, <b>J. Zhu</b>, ..., C. Sabatti, J. S. Boehm, W. C. Hahn, G. X.Y. Zheng, M. M. Davis, C. J. Kuo, <a href="https://www.sciencedirect.com/science/article/pii/S0092867418315137">&ldquo;Organoid modeling of the tumor immune microenvironment,&rdquo;</a>
                <i>Cell</i>, Vol. 175, No. 7, pp. 1972-1988, 2018.
                </p>
                </li>
                <li><p>
                W. C. Peng, C. Y. Logan, M. Fish, T. Anbarchian, F. Aguisanda, A. Alvarez-Varela, P. Wu, Y. Jin, <b>J. Zhu</b>, B. Li, M. Grompe, B. Wang, R. Nusse, <a href="https://www.sciencedirect.com/science/article/pii/S0092867418315046">&ldquo;Inflammatory cytokine TNFα promotes the long-term expansion of primary hepatocytes in 3D culture,&rdquo;</a>
                <i>Cell</i>, Vol. 175, No. 6, pp. 1607-1619, 2018.
                </p>
                </li>
                <li><p>B. Wang<sup>&dagger;</sup>, A. Pourshafeie<sup>&dagger;</sup>, M. Zitnik<sup>&dagger;</sup>, <b>J Zhu</b>, C. D. Bustamante, J. Leskovec, S. Batzoglou,
                <a href="https://arxiv.org/abs/1805.03327">&ldquo;Network Enhancement: a general method to denoise weighted biological networks,&rdquo;</a>
                <i>Nature Communications</i>, Vol. 9, No. 1, pp. 3108, 2018.
                </p>
                <li><p>B. Wang, D. Ramazzotti, L. De Sano, <b>J. Zhu</b></b>, E. Pierson, S. Batzoglou,
                <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/pmic.201700232">&ldquo;SIMLR: A Tool for Large-Scale Genomic Analyses by Multi-Kernel Learning,&rdquo;</a>
                <i>Proteomics</i>, Vol. 18,  No. 2, pp. 1700232, 2018.
                </p>
                <li><p>Y. Cao<sup>&dagger;</sup>, <b>J. Zhu</b><sup>&dagger;</sup>, G. Han, P. Jia, Z. Zhao
                <a href="http://www.mdpi.com/2073-4425/8/12/368/html">&ldquo;scRNASeqDB: a database for gene expression profiling in human single-cell by RNA-seq,&rdquo;</a>
                <i>Genes</i>, Vol. 8, No. 12, pp. 368, 2017.
                </p>
                </li>
                <li><p>B. Wang, <b>J. Zhu</b>, E. Pierson, D. Ramazzotti, S. Batzoglou,
                <a href="https://www.nature.com/articles/nmeth.4207">&ldquo;Visualization and analysis of single-cell RNA-seq data by kernel-based similarity learning,&rdquo;</a>
                <i>Nature Methods</i>, Vol. 14, No. 4, pp. 414, 2017.
                </p>
                </li>
                <li><p>
                G. X.Y. Zheng, J. M Terry, P. Belgrader, P. Ryvkin, Z. W. Bent, R. Wilson, S. B. Ziraldo, T. D. Wheeler, G. P. McDermott, <b>J. Zhu</b>, ..., T. S. Mikkelsen, B. J. Hindson, J. H. Bielas, <a href="https://www.nature.com/articles/ncomms14049">&ldquo;Massively parallel digital transcriptional profiling of single-cells,&rdquo;</a>
                <i>Nature Communications</i>, Vol. 10, No. 11, pp. 1096-1098, 2017.
                </p>
                </li>
                <li><p>B. Wang, <b>J. Zhu</b>, Oana Ursu, Armin Pourshafeie, Serafim Batzoglou, Anshul Kundaje,
                <a href="http://papers.nips.cc/paper/6290-unsupervised-learning-from-noisy-networks-with-applications-to-hi-c-data">&ldquo;Unsupervised learning from noisy networks with applications to Hi-C data',&rdquo;</a>
                in Proceedings of <i>Advances in Neural Information Processing Systems (NIPS)</i>, Barcelona, Dec. 2016.
                </p>
                </li>
                <!-- <li><p>
                <a href="">&ldquo;,&rdquo;</a>
                <i></i>.
                </p>
                </li> -->
                <li><p><b>J. Zhu</b>, S. Govindasamy, J. Hwang, <a href="http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&amp;arnumber=6825859&amp;isnumber=6880875">&ldquo;Performance of multiantenna linear MMSE receivers in doubly stochastic networks,&rdquo;</a> <i>IEEE Transactions on Communications</i>, Vol. 62, No.8, pp.2825,2839, Aug. 2014. </p>
                </li>
                <li><p><b>J. Zhu</b>, R. H. Y. Louie, M. R. McKay, S. Govindasamy, &ldquo;On the impact of unsynchronized interferers on multi-antenna OFDM systems,&rdquo; in Proceedings of  <i>the IEEE International Conference on Communications (ICC)</i>, Sydney, June 2014.</p>
                </li>
                <li><p><b>J. Zhu</b>, S. Govindasamy, <a href="http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&amp;arnumber=6655046&amp;isnumber=6654691">&ldquo;Performance of multi-antenna linear MMSE receivers in the presence of clustered interferers,&rdquo;</a> in Proceedings of <i>the IEEE International Conference on Communications (ICC)</i>, Budapest, June 2013. </p>
                </li>
                <li><p><b>J. Zhu</b>, S. Govindasamy, <a href="http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&amp;arnumber=6364332&amp;isnumber=6363631">&ldquo;Performance of multi-antenna MMSE receivers in non-homogenous Poisson networks,&rdquo;</a> in Proceedings of <i>the IEEE International Conference on Communications (ICC)</i>, Ottawa, June 2012. </p>
                </li>
                <li><p>N. Karst, J. Oehrlein, D. S. Troxell,  <b>J. Zhu</b>, &ldquo;The minimum span of L(2,1)-labelings of generalized flowers,&rdquo;
                <i>Discrete Applied Mathematics</i>, Vol. 181, pp. 139-151, January 2015.</p>
                </li>
                <li><p>N. Karst, J. Oehrlein, D. S. Troxell, <b>J. Zhu</b>, <a href="http://www.sciencedirect.com/science/article/pii/S0166218X14002972">&ldquo;Labeling amalgamations of Cartesian products of complete graphs with a condition at distance two,&rdquo;</a>
                <i>Discrete Applied Mathematics</i>, Vol. 178, pp. 101-108, December, 2014.</p>
                </li>
                <li><p>N. Karst, J. Oehrlein, D. S. Troxell, <b>J. Zhu</b>, <a href="***">&ldquo;On distance labelings of amalgamations and injective labelings of general graphs,&rdquo;</a>
                <i>Involve, a Journal of Mathematics</i>, Vol. 8, No. 4, pp. 535-540, 2014.</p>
                </li>
                <li><p>N. Karst, J. Oehrlein, D. S. Troxell, <b>J. Zhu</b>, <a href="http://link.springer.com/article/10.1007/s10878-013-9632-x">&ldquo;L(d, 1)-labelings of the edge-path-replacement by factorization of graphs.&rdquo;</a>
                <i>Journal of Combinatorial Optimization</i>, May 2013.</p>
                </li>
                <li><p>S. S. Adams, N. Howell, N. Karst, D. S. Troxell and <b>J. Zhu</b>, <a href="http://www.sciencedirect.com/science/article/pii/S0166218X12004313">&ldquo;On the L(2, 1)-labelings of amalgamations of graphs,&rdquo;</a> <i>Discrete Applied Mathematics</i>, Vol. 161, No. 7–8, pp.881-888, May 2013</p>
                </li>
                </ol>
            </div>
        </div>
    </div>
  </div>
  <script type="text/javascript">
    $('.panel-collapse').on('shown.bs.collapse', function (e) {
      var $panel = $(this).closest('.panel');
      $('html,body').animate({
          scrollTop: $panel.offset().top
      }, 500);
    });
  </script>
</body>
</html>
